backend/capellacollab/settings/integrations/purevariants/routes.py

```diff
@@ -5,20 +5,22 @@
 
 import typing as t
 
-from fastapi import APIRouter, Depends
+import fastapi
```

Comment:
Why import fastapi, and not import from fastapi, as we do in all other modules?

Reply:
```zsh
Use import statements for packages and modules only, not for individual classes or functions.
```

This is a rule in the Google styleguide and the team agreed to follow the Google styleguide. I personally also prefer the module imports, but if you don't agree, feel free to start a discussion in our cookiecutter repository: https://github.com/DSD-DBS/cookiecutter-dbs-python/discussions